### DISCUSSION TITLE
switch the page turning mode of the mouse wheel

### DISCUSSION THREAD

**Discussion by Qysrolan** [sumatrapdfreader/sumatrapdf#2709] [Q&A]
How do I switch the page turning mode of the mouse wheel, whether to scroll by row or the whole page

**Comment by GitHubRulesOK** [comment]
Mouse actions are generally controlled by the driver so you can use different drivers (with per app settings)
However native windows allows for such basic setting here on my laptop it is
![image](https://user-images.githubusercontent.com/12754913/173085064-5482642d-f5a2-4886-9db9-956daeee2f14.png)

One way to set scroll in pages would be to click the fit by page icon 
![image](https://user-images.githubusercontent.com/12754913/173083585-28a202f9-bfb2-41b5-88fa-a6b955e0a4d2.png)

Or set a key or mouse button to scroll up/down per page but that has limits and ramifications.

**Reply by ysrolan** [reply]
Thank you very much for your help, but I don't want to set it in windows. What I want to achieve is to use the mouse wheel to scroll the whole page in facing mode in Sumatra. I set the default facing mode, but I found that when I open different files, sometimes the mouse can scroll the whole page, and sometimes it scrolls by line, why this difference, can I set this by myself?

**Reply by GitHubRulesOK** [reply]
it will be relative to page mode 
so to force it would be `ctrl + 6+7` 
you could set opening preference to facing fit page (or content) but history if remembered will be last view
![image](https://user-images.githubusercontent.com/12754913/173244547-b9e45e27-3f5b-4ec1-8607-bfb80e43d429.png)

**Reply by ysrolan** [reply]
Thank you very much for your help, please, I have tried and this is still not the effect I want.
